### DISCUSSION TITLE
Add alerting to state timeline graph

### DISCUSSION THREAD

**Discussion by vdejager** [grafana/grafana#40610] [Alerting]
I'm using alerting on time series (packet loss) to detect flaws in our long range wireless connection to remote sensors in the field. Works great, but I would love to see alerting on state timeline graphs as well. Same for Stats graph etc.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
